RecoLocalCalo/HcalRecProducers/python/hbheRecHitProducerPortableTask_cff.py

```diff
@@ -0,0 +1,57 @@
+import FWCore.ParameterSet.Config as cms
+
+# Run 3 HCAL workflow on GPU
+
+# EventSetup modules used by HBHERecHitProducerPortable
+
+from RecoLocalCalo.HcalRecProducers.hcalMahiConditionsESProducer_cfi import hcalMahiConditionsESProducer
+from RecoLocalCalo.HcalRecProducers.hcalMahiPulseOffsetsESProducer_cfi import hcalMahiPulseOffsetsESProducer
+from RecoLocalCalo.HcalRecProducers.hcalSiPMCharacteristicsESProducer_cfi import hcalSiPMCharacteristicsESProducer 
+from RecoLocalCalo.HcalRecAlgos.hcalRecoParamWithPulseShapeESProducer_cfi import hcalRecoParamWithPulseShapeESProducer 
+
+hcalMahiPulseOffSetAlpakaESRcdSource = cms.ESSource('EmptyESSource',
+    recordName = cms.string('JobConfigurationGPURecord'),
+    iovIsRunNotTime = cms.bool(True),
+    firstValid = cms.vuint32(1)
+)
+
+# convert the HBHE digis into SoA format
+from EventFilter.HcalRawToDigi.hcalDigisProducerPortable_cfi import hcalDigisProducerPortable as _hcalDigisProducerPortable
+hcalDigisPortable = _hcalDigisProducerPortable.clone(
+    digisLabelF01HE = "f01HEDigisGPU",
+    digisLabelF5HB  = "f5HBDigisGPU",
+    digisLabelF3HB  = "f3HBDigisGPU"
```

Comment:
I would drop "GPU" from the labels, as the name of the module is enough to identify source of the collection.
Also, "GPU" is not correct, since this module may be running on cpu.

Reply:
Updated in 715105876f8999402d3e30f2fa94579275479978